testing/scenario_app/android/gradle.properties

```diff
@@ -1,4 +1,4 @@
-org.gradle.jvmargs=-Xmx1536M
+org.gradle.jvmargs=-Xmx2048M
```

Comment:
What's this for?

Reply:
I was just running into it.  I'm not 100% sure it caused by this change.   My theory is that having those extra .so files in the flutter.jar file somehow requires more memory when the build is happening.

Reply:
(this increases the JVM heap that gradle uses. It's probably fine to increase this)